server/src/test/java/org/elasticsearch/search/query/QueryStringIT.java

```diff
@@ -250,19 +250,18 @@ public void testPhraseQueryOnFieldWithNoPositions() throws Exception {
     }
 
     public void testBooleanStrictQuery() throws Exception {
-        Exception e = expectThrows(Exception.class, () ->
-                client().prepareSearch("test").setQuery(
-                        queryStringQuery("foo").field("f_bool")).get());
-        assertThat(ExceptionsHelper.detailedMessage(e),
+        Exception e = expectThrows(Exception.class,
+                () -> client().prepareSearch("test").setQuery(queryStringQuery("foo").field("f_bool")).get());
+        System.out.println(e);
```

Comment:
Is there any reason we are using `System.out.println` here? Or did you use for debugging?

Reply:
No, of course not. Thats from debugging and Checkstyle in CI should rightfully complain. Will remove this.